assets/js/data/tsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
   "extends": "../../../tsconfig.base.json",
   "compilerOptions": {},
-  "include": [ ".", "../type-defs" ],
+  "include": [ "." ],
```

Comment:
Don't we need to keep the `../type-defs` inclusion here for the type check? I'll add it back in but limit it to `.ts` files only in that folder and that way we're not inflating the count due to the js files in the folder.